src/main/ui/app/component/SearchBox.jsx

```diff
@@ -20,68 +21,50 @@ class SearchBox extends Component {
     }
 
     setQuery(query) {
-        this.setState({ query: query }, function() {
-            this.parseQuery();
-        })}
-
-    setSplitQuery(splitQuery) {
         this.setState({
-            splitQuery: splitQuery
+            query: query
         });
     }
 
-    parseQuery(){
-        if (this.state.query){
-            let advancedQuery = this.state.query;
+    doSearch() {
+        if (this.state.query) {
+            let advancedQuery = this.state.query.toLowerCase();
             //Basic Regex Matching to fix query inconsistencies
             //Removing all extra spaces i.e. two or more
             advancedQuery = advancedQuery.replace(/\s+/g,' ').trim();
             //Removing all spaces to the left of search keys
             advancedQuery = advancedQuery.replace(new RegExp("\\s+:","g"),":");
             //Removing all spaces to the right of search keys
             advancedQuery = advancedQuery.replace(new RegExp(":\\s+","g"),":");
-            let queryComponents = {};
-            //Determining query type
-            if(advancedQuery.includes(":")){
-                let querySplit = advancedQuery.split(' ');
-                let tempQuery = "";
-                querySplit.forEach(m => {
-                    if (m.includes(":")){
-                        let [key, value] = m.split(':');
-                        if (key === "type"){
-                            queryComponents["type"] = value.toUpperCase();
+            let url = "/search?";
+            // Check if query is an advanced query. We want to use the Parser only if the query is advanced
+            if (advancedQuery.includes(":") || advancedQuery.includes("-")){
```

Comment:
I think we want a way for the user to include a : or a - in the query. Can we document what it is in the help text? (Maybe not even so important to support this.)

Reply:
So do you mean to say that every query should have an advanced query like structure? IMO, certain users just prefer searching for text instead of having to worry about the structure of the advanced query and I believe we should continue to support that. Those who wish to go advanced will also have the option to do so. However, should you feel that we should do away with a simple query, do let me know and I will implement it :)